README.md

```diff
@@ -1,29 +1,123 @@
+
+Practical Example:
+
+User: `I would like to run http-minimum module.`
+
+```shell
+mvn clean verify -Dall-modules -Dnative -Dopenshift -pl http/http-minimum 
+```
+
+Quarkus test framework will reuse the Native binary generated by Maven to run the test, except if the scenario provides a build property, then it will generate a new Native binary.
+
+More info about how to generate native images could be found in Quarkus [building-native-image](https://quarkus.io/guides/building-native-image) website, and customize your native image generation by adding some [configuration flags](https://quarkus.io/guides/building-native-image#configuration-reference) to your maven command.
+
+For example, 
+
+User: `I want to use a custom GraalVM Docker image and 5GB of memory`
+
+```shell
+mvn clean verify -Dall-modules -Dnative -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-native-image:21.3-java11 -Dquarkus.native.native-image-xmx=5g -Dopenshift -pl http/http-minimum 
+```
+
+Or what about use the GraalVM that I have installed in my local environment instead of use a Docker image with GraalVM
+
+Be sure that GraalVM is installed by running the following command otherwise you will need to install it
+`gu --version`
+`gu install native-image`
+
+And then type this example
+
+User: `I want to use my local GraalVM in order to build my binary application`
+
+```shell
+mvn clean verify -Dall-modules -Dquarkus.package.type=native -Dopenshift -pl http/http-minimum 
+```
+
+### Bare metal
+
+Firstly be sure that docker/podman is running
+
+```shell
+docker run hello-world
+```
+
+Then just run a maven clean verify command pointing to your scenario. 
+
+Practical Example:
+
+User: `I would like to run http-minimum module.`
+
+```shell
+mvn clean verify -Dall-modules -pl http/http-minimum 
+```
+
+#### Bare metal / Native
+
+Same as [Openshift/native](#Openshift-/-Native) scenarios, Quarkus test framework will reuse the Native binary generated by Maven to run your tests.
+
+Practical Example:
+
+User: `I would like to run http-minimum module.`
+
+```shell
+mvn clean verify -Dall-modules -Dnative -pl http/http-minimum 
+```
+
+All the above example for openshift are also valid for Bare metal, just remove the flag `-Dopenshift` and play with [native image generation properties](https://quarkus.io/guides/building-native-image#configuration-reference)
+
+### Additional notes
+
+I recommend you to have a look the main `pom.xml` and pay attention to some useful topics as how the scenarios are categorized by topics or profiles, or some global properties as `quarkus.platform.version` that could be overwritten by a flag. 
+
+For example
+
+As a user I would like to run all core modules of Quarkus `2.2.3.Final`
+
+```shell
+mvn clean verify -Dquarkus-core-only -Dquarkus.platform.version=2.2.3.Final 
+```
```

Comment:
I don't understand this sentence :/
Does it applies to QuarkusTestSuite? There is no `quarkus-core-only` in the main pom.xml